presto-main/src/main/java/com/facebook/presto/sql/planner/sanity/PlanCheckerProviderManager.java

```diff
@@ -61,9 +62,11 @@ public void addPlanCheckerProviderFactory(PlanCheckerProviderFactory planChecker
         }
     }
 
-    public void loadPlanCheckerProviders()
+    public void loadPlanCheckerProviders(NodeManager nodeManager)
```

Comment:
Can NodeManager be injected instead along with SimplePlanFragmentSerde?

Reply:
We would need to bind the `PluginNodeManager` and @tdcmeehan had requested to not bind, but just create when needed. I agree since we already bind an internal node manager, this is just created from that to pass to plugins.